modules/twinklearv.js

```diff
@@ -247,20 +247,20 @@ Twinkle.arv.callback.changeCategory = function (e) {
 				type: 'textarea',
 				label: 'Evidence:',
 				name: 'evidence',
-				tooltip: 'Enter your evidence. It should make clear that each of these users is likely to be abusing multiple accounts. Usually this means diffs, page histories or other information that justifies why the users are a) the same and b) disruptive. This should purely be evidence and information needed to judge the matter. Avoid all other discussion that is not evidence of sockpuppetry or other multiple account abuse.'
+				tooltip: 'Your evidence should make it clear that each of these users is likely to be abusing multiple accounts. Usually this means diffs, page histories or other information that justifies why the users are a) the same and b) disruptive. This should purely be evidence and information needed to judge the matter. Avoid all other discussion that is not evidence of sockpuppetry.'
 			});
 			work_area.append({
 				type: 'checkbox',
 				list: [
 					{
-						label: 'Request CheckUser evidence',
+						label: 'Request CheckUser',
 						name: 'checkuser',
-						tooltip: 'CheckUser is a tool used to obtain technical evidence related to a sock-puppetry allegation. It will not be used without good cause, which you must clearly demonstrate. Make sure your evidence explains why CheckUser is appropriate.'
+						tooltip: 'CheckUser is a tool used to obtain technical evidence related to a sockpuppetry allegation. It will not be used without good cause, which you must clearly demonstrate. Make sure your evidence explains why using the tool is appropriate. It is only used to connect user accounts and not IP addresses.'
```

Comment:
>It is only used to connect user accounts and not IP addresses.
Agree that it's good to say this, but isn't this a little vague?  Couldn't it imply to someone unfamiliar with it that it won't work on IP addresses?  Maybe something like: `It will not be used to publicly connect user accounts and IP addresses.`